server/events/project_command_builder_internal_test.go

```diff
@@ -653,6 +653,183 @@ projects:
 	}
 }
 
+func TestBuildProjectCmdCtx_Logging(t *testing.T) {
```

Comment:
Logging is a little misleading imo. Can we change it to something else? Maybe `TestBuildProjectCmdCtx_LogLevel`?